tests/test_fortune.py

```diff
@@ -0,0 +1,17 @@
+from sarathi.til import print_fortune
+
+def test_fortune():
+    ''' Tests the Fortune Command '''
+    k,l = print_fortune()
+    if l:
+        print('--------------------Faiure--------------------------')
+    # l = str(k)
+    print(k)
+    
+    # with open('/home/filius-fall/fortunes/fortunes.txt','r') as file:
+    #     for line in file:
+    #         if k in line:
+    #             print('------------------Successs----------------------------')
+    #         else:
+    #             print('--------------------Failure----------------------------')
+    assert k
```

Comment:
To get you started, I am going to write the test for you. Try to write the function that makes it run.

```python

def test_call_fortune():
    """Tests that the `call_fortune` function in the `fortune` module works as planned."""
    from sarathi.fortune import call_fortune
    fortune_string = call_fortune()
    assert isinstance(fortune_string, str), "The call_fortune function should have returned a string."

```

That's it, really. You don't need anything else.